src/loader.c

```diff
@@ -996,6 +996,11 @@ static void ldr_load_pw_line(struct db_main *db, char *line)
 
 		if (!(db->options->flags & DB_WORDS) && dupe_checking) {
 			int collisions = 0;
+			int hash_collisions_max = LDR_HASH_COLLISIONS_MAX;
+
+			if (!format->params.binary_size)
+				hash_collisions_max *= 10;
```

Comment:
I think it'd be cleaner to have this x10 multiplier or alternative max value in `params.h` near `LDR_HASH_COLLISIONS_MAX`. Also, why was 1000 not sufficient but 10000 expected to be sufficient? 1000 was kind of arbitrary, so if only a 10x increase is needed we can simply bump it to 10000.

Reply:
> I think it'd be cleaner to have this x10 multiplier or alternative max value in `params.h` near `LDR_HASH_COLLISIONS_MAX`. Also, why was 1000 not sufficient but 10000 expected to be sufficient? 1000 was kind of arbitrary, so if only a 10x increase is needed we can simply bump it to 10000.

Perhaps just muting them with no limit would be more logical? The formats can't be fixed anyway.

If not, I agree just bumping LDR_HASH_COLLISIONS_MAX for now is better bc KISS.

Reply:
> Perhaps just muting them with no limit would be more logical? The formats can't be fixed anyway.

Maybe a problem with that is the user won't get an explanation about load time (which I presume increases non linearly).